app/AccountancyModule/PaymentModule/presenters/BankAccountsPresenter.php

```diff
@@ -236,7 +236,7 @@ private function assertCanViewBankAccount(int $id): void
         if (
             $role->getUnitId() === $account->getUnitId()
             && $role->isBasicUnit()
-            && ($role->isAccountant() || $role->isOfficer() || $role->isEventManager())
+            && ($role->isAccountant() || $role->isOfficer() || $role->isEventManager() || $role->isEducationAccountant())
```

Comment:
a proc to nepripojit pod funkci `role->isAccountant()` a dělat na to samostatnou? 

A nenastane v obou případech, že hospodář VzA dostane přístup k uctu i jednotky jako takové?

Reply:
Jo, určitě by to šlo připojit pod `isAccountant()`, ale přišlo mi lepší nemíchat 2 různé role do jedné funkce, zvlášť když ta vnitřní logika těch rolí je úplně jiná.

Ano, hospodář VzA tím dostane přístup i k platbám jednotky. Mně to přijde asi v pohodě, z důvodů popsaných v těle PR.

Reply:
Za mě je to stále `Accountant`, tak bych to spíš spojil. Protože stejně jsme udělali `Officer` pres více úrovní a rolí.

S tím přístupem jsem si toho nejdřív nevšiml. Každopádně bych přidal i hlášku do plateb, že přístup k platbám mají i role VzA pod danou jednotkou.

Reply:
Ok, sloučil jsem ty funkce. Co myslíš hláškou? Permanentní baner mi přijde jako overkill...

Reply:
Jako hospodar, co si zadá ucet do h.skautingu neocekávám, že se k tomu automaticky dostane i vedoucí vzdělávačky. Pokud existuje nějaký precedens pro přístup lidí z VzA k jednotce, tak to udělat podle toho @manulekonecna

Reply:
Tak jako precedens je podle mě trochu v cesťácích - tam taky s rolí vzdělávačky vidím i cesťáky a auta střediska...

Napadá mě ještě tam ten baner dát a třeba za 3 měsíce ho zase zrušit. Klidně na to připravím druhé PR...

Reply:
jeste bych rád pohled @manulekonecna nebo @jerrysohn

Reply:
@manulekonecna ?

Reply:
Jen ještě pro kontext doplním, jaké je typické řešení tohohle problému ze strany pořádajících jednotek - hospodář dostane v ISu roli správce akcí, potažmo vedoucí/admin... :D